exec.go

```diff
@@ -72,7 +87,18 @@ func (et ExecTask) Execute(ctx context.Context) (ExecResult, error) {
 	var command string
 	var commandArgs []string
 	if et.Shell {
-		command = "/bin/bash"
+
+		// On a NixOS system, /bin/bash doesn't exist at /bin/bash
```

Comment:
Correct, although the same is true for the *BSD systems, e.g. on OpenBSD bash (if installed) resolves to /usr/local/bin/bash.

Reply:
TIL